CentralityiOS/CentralityiOS/View Controllers/AddTaskModalViewController.h

```diff
@@ -0,0 +1,21 @@
+//
+//  AddTaskModalViewController.h
+//  CentralityiOS
+//
+//  Created by Samuel Osa-Agbontaen on 7/11/22.
+//
+
+#import <UIKit/UIKit.h>
+#import "TaskObject.h"
+
+@class AddTaskModalViewController;
+
+@protocol AddTaskModalViewControllerDelegate <NSObject>
+- (void)addNewTaskToFeed:(AddTaskModalViewController *)controller newTaskToAddToFeed:(TaskObject *)item;
```

Comment:
By convention and for readability, we usually name a delegate method by `did<action>` so here you can name it to something like `didAddNewTask:(TaskObject *)item toFeed:(AddTaskModalViewController *)controller`.